compiler/rustc_feature/src/unstable.rs

```diff
@@ -193,6 +193,8 @@ declare_features! (
     (unstable, anonymous_lifetime_in_impl_trait, "1.63.0", None),
     /// Allows identifying the `compiler_builtins` crate.
     (internal, compiler_builtins, "1.13.0", None),
+    /// Gating for a new desugaring of const arguments of usages of const parameters
+    (internal, const_arg_path, "1.81.0", None),
```

Comment:
I just put this as internal since it's only really being added to avoid a beta regression and is not really intended to be a "real" feature since it has no user impact whatsoever